source/common/network/apple_dns_impl.cc

```diff
@@ -71,9 +71,16 @@ void AppleDnsResolverImpl::initializeMainSdRef() {
 }
 
 void AppleDnsResolverImpl::onEventCallback(uint32_t events) {
-  ENVOY_LOG(debug, "DNS resolver file event");
+  ENVOY_LOG(debug, "DNS resolver file event ({})", events);
   ASSERT(events & Event::FileReadyType::Read);
-  DNSServiceProcessResult(main_sd_ref_);
+  DNSServiceErrorType error = DNSServiceProcessResult(main_sd_ref_);
```

Comment:
I verified that _all_ other calls to the DNSService API that return `DNSServiceErrorType` are handled.